lib/blinds_lib.py

```diff
@@ -409,11 +423,8 @@ def Up(self, message):
       return self.SetDesiredPositions(self.UP, self.UP, message)
 
   def Control(self):
-    if self.alarm_lock == True and self.wind_lock != True:
-      pass
-      # return self.Down(self.DOWN, self.DOWN, 'Alarm lock is on. Nobody is home. Blinds down')
     if self.wind_lock == True:
-      return self.DoNothing('Wind lock is on')
+      return self.SetDesiredPositions(self.UP, self.UP, 'Wind Alarm blinds go up to prevent damage')
```

Comment:
hm... that's not ideal. in my case it's not home assistant that sends up the blinds, it's the wind sensor that tells the blinds directly to go up. with this, the blinds also go up, but what happens if the wind_lock is False again, will it go down after that?